nixos/modules/services/web-apps/wakapi.nix

```diff
@@ -134,6 +188,10 @@ in
         assertion = cfg.smtpPassword != null -> cfg.smtpPasswordFile != null;
         message = "Both `services.wakapi.smtpPassword` `services.wakapi.smtpPasswordFile` should not be set at the same time.";
       }
+      {
+        assertion = cfg.db.createLocally -> cfg.db.dialect != null;
+        message = "`services.wakapi.database.createLocally` is true, but a database dialect is not set!";
```

Comment:
maybe add some note about only postgres being supported here too?

Reply:
that would also do, the current message implies that you can pick any dialect and `createLocally` will do something

Reply:
Though null already covers "not postgres". I'm not sure, maybe it's safer to just mention we only support postgresql.

Reply:
I've added a warning to get around this. It's probably not worth an assertion anyway.